src/App.js

```diff
@@ -9,7 +11,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1 className="app-title">React Article</h1>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
+    />
```

Comment:
There is easier way to pass all object fields to props:
```suggestion
    <Article {...article} />
```